src/App/Patterns/content/Forms/constants.js

```diff
@@ -52,7 +52,7 @@ export const formsShowCase = {
             tab: "Form",
             component:
             <div className="col-md-10 col-12 px-0">
-                <form>
+                <form className="px-4">
```

Comment:
This is pretty interesting.. When removing this class in stage env it looks good, but if I do it in local v, it looks awful 🤔 Is there a magical margin that are only included when working locally?

Reply:
ooooh, that's weird 🤔
but, I'm not sure to understand what you're saying. For me I see the same UI changes on local and stage when I have and remove `px-4`. With the same horizontal overflow for the "multi-step form" tab.

also, I feel like we do not need it actually ? padding is good enough without the `px-4`
(I added it because at first I did not had the automatic external padding, but I added it later, so this px-4 is probably redundant now 🧐 )

Reply:
Agreed, it can be removed :D This comment was written when I was still on the wrong branch locally 😂